webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -15113,6 +15093,38 @@ var userOverride = {
                     $file_progress.removeClass('n-vis').addClass('vis');
                 }
             };
+            _this.handleEncryptedElements = function(encryptedElements){
+                for(var i = 0; i < encryptedElements.length; i++){
+                    switch (encryptedElements[i].tagName) {
+                        case 'IMG':
+                            var img = encryptedElements[i];
+                            if (KommunicateUI.isInView(img, document.querySelector('#mck-message-cell .mck-message-inner'))) {
+                                KommunicateUI.processLazyImage(img, img.getAttribute('data-thumbnailBlobKey'))
+                            }
+                            break;
+                        case 'AUDIO':
+                            var video = encryptedElements[i];
+                            if (KommunicateUI.isInView(video, document.querySelector('#mck-message-cell .mck-message-inner'))) {
+                                KommunicateUI.processEncMedia(video, video.getAttribute('data-blobkey'))
+                            }
+                            break;
+                        case 'VIDEO':
```

Comment:
whats the difference in Audio and video? it looks to be sae

Reply:
no there was no difference.
Previously I didn't knew how to use two values for one switch case statement
Have update the code now.
Thanks